cloudevents/tests/test_http_events.py

```diff
@@ -144,11 +148,10 @@ def test_emit_structured_event(specversion):
         "specversion": specversion,
         "data": test_data
     }
-    event = CloudEvent(body, headers=headers)
```

Comment:
See the argument above -- I don't think we want to round-trip the CloudEvent on both sides because it makes it harder to test the protocol conformance (though the code would happily be much simpler).